modules/light-clients/06-solomachine/types/update_test.go

```diff
@@ -615,7 +623,7 @@ func (suite *SoloMachineTestSuite) TestUpdateState() {
 						suite.Require().Equal(consensusState, cs.(*types.ClientState).ConsensusState)
```

Comment:
can you add checks on the successful case that the correct client state is set in the store?